test/utils/integration/integration.go

```diff
@@ -873,6 +941,37 @@ func (env TestEnvironment) GetServiceFromK8s(name, namespace string) (*corev1.Se
 	return result, nil
 }
 
+func (env TestEnvironment) GetSecretFromK8s(name, namespace string) (*corev1.Secret, error) {
+	nn := types.NamespacedName{
+		Name:      name,
+		Namespace: namespace,
+	}
+	result := &corev1.Secret{}
+	if err := env.k8sClient.Get(env.Context, nn, result); err != nil {
+		return nil, err
+	}
+	return result, nil
+}
+
+func (env TestEnvironment) GetMutatingAndValidatingWebHookConfigFromK8s(ctx context.Context) (
+	*admissionv1.MutatingWebhookConfiguration, *admissionv1.ValidatingWebhookConfiguration, error) {
+	var mutatingWH admissionv1.MutatingWebhookConfiguration
+	mutatingWHKey := client.ObjectKey{
+		Name: getTestBackendConfig().MutatingWebhookName,
+	}
+	if err := env.k8sClient.Get(ctx, mutatingWHKey, &mutatingWH); err != nil {
+		return nil, nil, err
+	}
+	var validatingWH admissionv1.ValidatingWebhookConfiguration
+	validatingWHKey := client.ObjectKey{
+		Name: getTestBackendConfig().ValidatingWebhookName,
+	}
+	if err := env.k8sClient.Get(ctx, validatingWHKey, &validatingWH); err != nil {
+		return nil, nil, err
+	}
+	return &mutatingWH, &validatingWH, nil
+}
```

Comment:
this one seems to me duplicate as you have this `func (r *Reconciler) getMutatingAndValidatingWebHookConfig(ctx context.Context)` in webhook.go file too. Let's create a common function in KubeClient?

Reply:
updated it to use the common methods from `KubeClient`.